params/chris_bird.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.677
+    
+# e.g. "anas_krystallinus"    
+species.name <- "green_chicken"
```

Comment:
Try not to use underscores in your species name, this would be better:
```suggestion
species.name <- "Green Chicken"
```